.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     steps:
       - checkout
       - run: git submodule update --init --recursive
+      - run: apt-get update && apt-get -y install python3.6 && apt -y install python3-pip && pip3 install conan
```

Comment:
This CI job was broken because of a compilation error in the old version of `conan` that comes with [zondax docker image](https://github.com/Zondax/docker-circleci/blob/master/Dockerfile#L26-L27). 
The newer versions of `conan` require `python3` and solve the issue.

Reply:
Thank you for debugging this 🙏